base.yml

```diff
@@ -4,7 +4,7 @@ services:
     image: danpoltawski/moodle-php-apache:7.1
     depends_on:
       - db
-      - mail
+      - mailhog
```

Comment:
Can we put this into `services.mail.yml`? It's a bit unclean in my mind to depend on something which isn't included in the same file.